### PR TITLE
Fix dispatcher - WIP

### PR DESCRIPTION
**Pull request by DarthMazut**

### CODE REVIEW

**Review by FWidel — COMMENTED** [review]
f

File: ViewModels/DispatchingPageViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    public partial class DispatchingPageViewModel : INavigationParticipant
+    {
+        public INavigator Navigator => MochaCore.Navigation.Navigator.Create();
+
+        [RelayCommand]
+        private void TestDispatcher()
+        {
+            IDispatcher dispatcher = DispatcherManager.GetMainThreadDispatcher();
+
+            Debug.WriteLine("Start TestDispacther()");
```
[the remark this answers is not in the document]
Comment (DarthMazut) [reply ViewModels/DispatchingPageViewModel.cs:22]: This is not UnitTest

File: ViewModels/DispatchingPageViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            dispatcher.RunOnMainThread(async () =>
+            {
+                Debug.WriteLine("Start executing async delegate");
+                await Task.Delay(5000);
+                //Thread.Sleep(5000);
```
[the remark this answers is not in the document]
Comment (DarthMazut) [reply ViewModels/DispatchingPageViewModel.cs:27]: Comments are good because they're describe what code do

**Comment by DarthMazut** [comment]
@FWidel thank you for professional review 👍

**Pull request merged**